File: python/uberAdvanced.py
from car import Car

class UberAdvanced (Car):
    typeCarAccepted = []
    seatsMaterial = []

    def __init__ (self, id, license, driver, passengers, typeCarAccepted, seatsMaterial) -> None:
        super().__init__(id, license, driver, passengers)

        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial

File: js/uberAdvanced.js
class UberAdvanced extends Car {
    #typeCarAccepted;
    #seatsMaterial;

    constructor (id, license, driver, passengers, typeCarAccepted, seatsMaterial) {
        super(id, license, driver, passengers);
        this.#typeCarAccepted = typeCarAccepted;
        this.#seatsMaterial = seatsMaterial;
    }

    getTypeCarAccepted () {
        return this.#typeCarAccepted;
    }

    setTypeCarAccepted (typeCarAccepted) {
        this.#typeCarAccepted = typeCarAccepted;
    }

    getSeatsMaterial () {
        return this.#seatsMaterial;
    }

    setSeatsMaterial (seatsMaterial) {
        this.#seatsMaterial = seatsMaterial;
    }
}